backend/index.ts

```diff
@@ -0,0 +1,17 @@
+import { App } from './src/server';
```

Comment:
We should avoid all these changes in the backend, backend shouldn't know anything about tests

Reply:
Backend doesn't know anything about tests. I just splitted one file into three: construction of server, listen function and call of listen function.